.github/workflows/test.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Build Dist 🔧
         run: yarn run build
 
-  chromatic-deployment:
+  storybook:
```

Comment:
Figured this job could be everything that needs storybook to be built (a11y tests, chromatic)

Reply:
Yes, appreciate the simplicity